news/2024/2024-01-28-vocaloid-mapping-contest-2.md

```diff
@@ -0,0 +1,185 @@
+| **Lyricist** | π |
+| **Download** | [Beatmap template (4:55 / 128 BPM)](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.osz) |
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.mp3" type="audio/mpeg">
+    </audio>
+</div>
+
+| Song | iroha(sasaki) - sigh |
+| :-- | :-- |
+| **Vocaloid** | Kagamine Rin |
+| **Lyricist** | iroha(sasaki) |
+| **Download** | [Beatmap template (4:08 / 148 BPM)](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.osz) |
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.mp3" type="audio/mpeg">
+    </audio>
+</div>
+
+### Rules
+
+- **Submissions must use one of the [selected songs](#songs).**
+- **Submissions must be in `.osz` format.**
+- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes (such as an accidental off-screen object), submissions that contain intentional unrankables (not due to simple error, such as sliderator) will be disqualified. Mappers will be notified if this occurs.
+- **Submissions must not be publicly revealed until after the results are announced.**
+- **Excluding metadata, submissions must not include anything that could reveal your identity.** Metadata will be anonymised before submissions are relayed to judges and screeners.
+- **Collaborations are not allowed.** This is a solo contest.
+- **Submissions must not include storyboards or videos.**
+- **Your submission must not have any edits to the `.osu` file that would not be possible through the editor.** This isn't Aspire. You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
+- **Your submission must have only one difficulty.**
+
+### Submission
+
+Once your beatmap is ready, head to the relevant contest listing to submit your map:
+
+- [osu! contest listing](https://osu.ppy.sh/community/contests/199)
+- [osu!taiko contest listing](https://osu.ppy.sh/community/contests/200)
+- [osu!catch contest listing](https://osu.ppy.sh/community/contests/201)
+- [osu!mania contest listing](https://osu.ppy.sh/community/contests/202)
+
+After the submission period closes, judging will begin. Find out more details about the judges and the judging criteria on the [contest's wiki page](/wiki/Contests/VMC/2).
+
+---
+
+And that should be everything you need to know to get started!
+
+For the most up-to-date info about the contest, keep the [**Vocaloid Mapping Contest** wiki page](/wiki/Contests/VMC/2) bookmarked. Feel free to also join the [VAT (Vocaloid Appreciation Team) Discord server](https://discord.com/invite/EruvDHaReY) to receive updates on the VMC (as well as future events), map feedback, info on collaborative VOCALOID mapping projects, and more!
+
+We look forward to seeing your maps!
+
+—-White
```

Comment:
Isn't pishifat responsible for FA outreach? Maybe add his name on that line if he did that?

Reply:
the author line is usually for the people who wrote the post, i.e. you in this case